src/backends/caffe2/netdef_backend.cc

```diff
@@ -86,8 +86,11 @@ ConvertDataType(DataType dtype)
 
 
 NetDefBackend::Context::Context(
-    const std::string& name, const int gpu_device, const int max_batch_size)
-    : BackendContext(name, gpu_device, max_batch_size)
+    const std::string& name, const int gpu_device, const int max_batch_size,
+    const bool enable_indirect_input, const bool enable_indirect_output)
```

Comment:
"indirect" seems unnecessarily vague. Can we just called it "enable_pinned_input", etc?